docs/make.jl

```diff
@@ -0,0 +1,23 @@
+using SciMLModelCoupling
+using Documenter
+
+DocMeta.setdocmeta!(SciMLModelCoupling, :DocTestSetup, :(using SciMLModelCoupling); recursive=true)
+
+makedocs(;
+    modules=[SciMLModelCoupling],
```

Comment:
Can't this information be taken from `Project.toml`?
If not, it might be good to add the other authors here too.

Reply:
I'm not sure I understand.

Reply:
I was referring mainly to the lines below (sorry for the wrong selection). When generating this docs, you have some metadata such as: authors, site name, that are also in the `Project.toml`. So my question was if we could retrieve that info from `Project.toml`. That said, these info doesn't change much and thus maybe it is simpler to have it duplicated.

Reply:
Let's stick now to packaging. I'll deal with documenting later.